data/publisher/src/main/java/tech/pegasys/teku/data/publisher/MetricsPublisherManager.java

```diff
@@ -35,7 +36,7 @@
   private final AsyncRunnerFactory asyncRunnerFactory;
   private final MetricsEndpoint metricsConfig;
   private final MetricsDataFactory dataFactory;
-  private final JsonProvider jsonProvider = new JsonProvider();
+  private final ObjectMapper mapper;
```

Comment:
Why did we stop going via `JsonProvider`?  It would be better if we can continue using it for consistency and to provide some abstraction over the underlying library.